VAMobile/src/screens/HealthScreen/Appointments/PastAppointments/PastAppointments.tsx

```diff
@@ -171,6 +165,19 @@ function PastAppointments({
   const pickerOptions = getPickerOptions()
   const [datePickerOption, setDatePickerOption] = useState(pickerOptions[0])
 
+  const filteredAppointments = useMemo(
+    () =>
+      datePickerOption.timeFrame === TimeFrameTypeConstants.PAST_THREE_MONTHS
+        ? filterAppointments(appointmentsData?.data || [], true)
```

Comment:
Won't we need to do this filtering for `PAST_ALL_CURRENT_YEAR` as well?

Reply:
Yes. Let me add now.